pubpub/pubpub
# update: Langing page for Platform

## Issue(s) Resolved
Updates our landing page to reflect PubPub Platform, and restricts new community creation to superAdmins.

## Test Plan
1. Load www on local or test repo and make sure the landing page loads as shown below.
2. Click on all the links and make sure they lead to the right pages on the knew kf.org
3. Attempt to create a community while logged in as a non-super-admin. You should get an error.
4. Log in as a superadmin and attempt to create a new community. It should succeed (note: it will forward you to community.pubpub.org, which will throw an error, but the community will be available at community.duqduq.org).

## Screenshots (if applicable)
<img width="1183" alt="Screenshot 2024-07-17 at 14 26 52" src="https://github.com/user-attachments/assets/5196184a-870f-47bd-92e5-15fdb8927dea">


## Optional

### Notes/Context/Gotchas

### Supporting Docs
